RobotLib/src/main/java/com/acmerobotics/library/dashboard/config/Option.java

```diff
@@ -0,0 +1,127 @@
+package com.acmerobotics.library.dashboard.config;
```

Comment:
what is the purpose of putting all this logic in here and using an enum? why not put in the subclasses
could you use this.class? idk if classes work that way

Reply:
Yeah I'm going to restructure it better. I don't think `this.class` works that way.